resources/config-defaults.edn

```diff
@@ -385,4 +385,6 @@
 
  :enable-catalogue-table true ; show catalogue page table of items
  :enable-catalogue-tree false ; show catalogue page tree of items
- :catalogue-tree-show-matching-parents true} ; should parent categories be shown if children match?
+ :catalogue-tree-show-matching-parents true ; should parent categories be shown if children match?
+ 
+ :enable-cart true} ; allow bundling multiple items to application
```

Comment:
There can be multiple resources in the shopping cart, and one can apply for them one at a time, in bundles or separately, depending on the case. So I would say here that "show shopping cart and allow bundling multiple resources into one application"